src/main/msp/msp.c

```diff
@@ -1369,41 +1369,80 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU16(dst, flight3DConfig()->deadband3d_throttle);
         break;
 
+
     case MSP_SENSOR_ALIGNMENT: {
         uint8_t gyroAlignment;
 #ifdef USE_MULTI_GYRO
         switch (gyroConfig()->gyro_to_use) {
         case GYRO_CONFIG_USE_GYRO_2:
-            gyroAlignment = gyroDeviceConfig(1)->align;
+            gyroAlignment = gyroDeviceConfig(1)->alignment;
             break;
         case GYRO_CONFIG_USE_GYRO_BOTH:
             // for dual-gyro in "BOTH" mode we only read/write gyro 0
         default:
-            gyroAlignment = gyroDeviceConfig(0)->align;
+            gyroAlignment = gyroDeviceConfig(0)->alignment;
             break;
         }
 #else
-        gyroAlignment = gyroDeviceConfig(0)->align;
+        gyroAlignment = gyroDeviceConfig(0)->alignment;
 #endif
         sbufWriteU8(dst, gyroAlignment);
         sbufWriteU8(dst, gyroAlignment);  // Starting with 4.0 gyro and acc alignment are the same
-        sbufWriteU8(dst, compassConfig()->mag_align);
+        sbufWriteU8(dst, compassConfig()->mag_alignment);
```

Comment:
why not `alignment` as all other sensors?

Reply:
Some of the variable names with `_` in them are supposed to directly correlate to the the strings for their setting in settings.c.  e.g. there should also be a cli setting called `mag_align`, however this is not the case here as has at some-point been renamed to from `align_mag` to `mag_align` in the code.

There is a general lack of consistency with regards to this naming approach, I never thought it was a good idea anyway since internal values should be decoupled from their external interface...

so in this case, I ditched the abbreviation of "align" for the internal name, as that's what it represents, we're not "align"ing, we're holding the value of an "alignment".

Reply:
I forget to add my conclusion, yes the `mag_` prefix could probably be removed, but that's a can of worms I'm not willing to open in this PR as there are so many placed that would have to be changed.